TypewiseAlert/EmailAlert.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+
+namespace TypewiseAlert
+{
+    class EmailAlert : IAlerter
```

Comment:
EmailAlert implementation is incomplete

Reply:
Email alert implementation takes breachType as parameter. Based on the breach type either low or high it takes from enum and displays. Do I need to add reflections to fetch the breachType. Please let me know sir. Thank you.

Reply:
Multiple solutions , either you can dictionary or reflection